api/types/events/oneof.go

```diff
@@ -843,6 +843,16 @@ func ToOneOf(in AuditEvent) (*OneOf, error) {
 		out.Event = &OneOf_AWSICResourceSync{
 			AWSICResourceSync: e,
 		}
+
+	case *WorkloadIdentityX509IssuerOverrideCreate:
+		out.Event = &OneOf_WorkloadIdentityX509IssuerOverrideCreate{
+			WorkloadIdentityX509IssuerOverrideCreate: e,
+		}
+	case *WorkloadIdentityX509IssuerOverrideDelete:
+		out.Event = &OneOf_WorkloadIdentityX509IssuerOverrideDelete{
+			WorkloadIdentityX509IssuerOverrideDelete: e,
+		}
+
```

Comment:
just curious why we dont emit event for the other actions? (update?)

Reply:
Historically we have called `create` what should've been `write`; there's no way to know whether or not an upsert operation is an update or a create, and adding a separate `update` event (and maybe even an `upsert` one) just leads to having to deal with more events for no real benefit, IMO. We have followed this pattern with other resources, from what I can tell.